src/components/Inbox/NewMessage.jsx

```diff
@@ -0,0 +1,175 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { useSession } from '@inrupt/solid-ui-react';
+import { sendMessageTTL } from '../../utils';
+
+/**
+ * New Message Component - Component that allows user to write
+ *  a message to another user from their inbox
+ *
+ * @memberof Inbox
+ * @name NewMessage
+ */
+
+const NewMessage = () => {
+  const { session } = useSession();
+  const [message, setMessage] = useState({
```

Comment:
Do we need both the `Recipient Username` and `Recipient Full Name`? 

Do we need the `Sender Name` at all?  Are we not able to get this from their account/solid/pod info?

------

If after answering these it is decided that changes are needed, then changes will be necessary for the error messages code too.

Reply:
I think could try and pull information from just the username's profile card.

Reply:
Created a new function in session-helper called getUserProfileName to fetch the sender and recipient's name from their usernames and attach it to the turtle file message.